variation-commons-core/src/main/java/uk/ac/ebi/eva/commons/core/models/VariantType.java

```diff
@@ -17,29 +17,46 @@
 
 /**
  * Type of variation, which depends mostly on its length.
+ * See https://docs.google.com/spreadsheets/d/1YH8qDBDu7C6tqULJNCrGw8uBjdW3ZT5OjTkJzGNOZ4E/edit#gid=1433496764
  */
 public enum VariantType {
 
     /**
      * SNVs involve a single nucleotide, without changes in length
```

Comment:
Isn't that in the docs?

Reply:
I was just thinking that making the code documentation self-contained would be useful, whereas the link would contain additional details. Otherwise it is not obvious that we are matching the sequence ontology.